### ISSUE TITLE
Add a .prettierrc (or .prettierrc.json) to auto-format JSON (and JS) files

### ISSUE THREAD

**Issue by atuttle** [foundeo/cfdocs#1027]
[Prettier](https://github.com/prettier/prettier#readme) is great. Having a definitions file will ensure that (at least for people who have the extension installed in their editor), the files are all consistently formatted.

**Comment by pfreitag** [comment]
Good idea -- if you want to submit a PR we can see how well it might work.

**Comment by atuttle** [comment]
Here's the config I would recommend, but it does incorporate some of my own personal preferences.
```
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "parser": "json",
  "printWidth": 80,
  "proseWrap": "preserve",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 4,
  "trailingComma": "none",
  "useTabs": true
}
```

[Here's an easy way to try the different options and pick the config you like](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEwA6ICGAbbB5KDJAAgwGUBLKAc2zgwBpSQAZK+5ZuAD0wFsADnUZpCIKBDIDMYDkhgAnAK5wGo4swCOSiPCLF0UDRoDkAZyq04ARR3wTJEz35C4JtUeNfv3jFQAmCPD++ooqHhoAvhHMAgAW0HAAckp8AEZwCmb6ANrqxuggMACeAnLMCXwcMV4YUKkZCvoYAEwAjC3EAKw9ALQdAMwALBjR+RqFJWXNIBAAZnMUsiKetUUKmBTYljOQfHyYZsRpxcQyYBSBsKMxALoekSAMswIwFNBmyKCYCgoQAO4ABR+CE+KEwADcIJcniA0hswABrOAwKQySzIMKqEBxGB8bAAdTiFHgZmksjIoJJFAhJOKyHAZk+zyoZkyMEBG2oB2QcxwbOeACszNwAEII5Go-hwNhQOC8-nY4XcSg0Oi2XTypB87ACkDSLKZBnC6CwgQKKgwAmXGBxZAADgADM9zRA2QSNgIGea4GyFBD5c8FHBtBRg5zMNzMArddi2XwKJjlHHLOq7FqdXqYJg0tb-LbkENnopNtsaABhCD7aMoCRy2FKNkAFRzYKxkUiQA). There's a "copy config json" button at the bottom once you've got settings you're happy with.

**Comment by pfreitag** [comment]
Cool - that looks good to me, @shaedrich @KamasamaK what do you think?

**Comment by KamasamaK** [comment]
I've never used Prettier, but it seems like a good idea to enforce (formatting) standards for the project.

**Comment by psarin** [comment]
Please see https://github.com/foundeo/cfdocs/pull/1649 for committed rules for .prettierrc and .cflintrc. If people are okay with these rules, I can apply to JSON files.
